src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -796,7 +814,7 @@ class module_elf : public module_impl
   ert_cmd_opcode
   get_ert_opcode() const override
   {
-    if (m_os_abi == Elf_Amd_Aie2ps)
+    if (m_os_abi == Elf_Amd_Aie2ps || m_os_abi == Elf_Amd_npu3)
```

Comment:
I am not an expert of this area of the code, but it looks to me that there are lots of code like
"if aie2ps or aie4(npu3) then ..." can we make it more clear if you don't want to use more comment to explain what we are doing here.
1) can we use aie2ps and aie4 instead of aie2ps and npu3, those seems not a pair in same level
2) can we use a inline function or macro to make those dup code less?